app/src/ui/history/commit-summary.tsx

```diff
@@ -157,13 +152,6 @@ export class CommitSummary extends React.Component<
     }
   }
 
-  private onHideWhitespaceInDiffChanged = (
-    event: React.FormEvent<HTMLInputElement>
-  ) => {
-    const value = event.currentTarget.checked
-    return this.props.onHideWhitespaceInDiffChanged(value)
-  }
```

Comment:
This was used in `Hide Whitespace Changes` when it was a checkbox inline in the diff header, but not needed when it was moved into the `Diff Options` popover.